stockflux-securities-master/src/services/SecuritiesService.js

```diff
@@ -85,3 +85,24 @@ export async function deleteSecurity(securityId) {
   const json = await response.json();
   throw new ValidationError(json.messages);
 }
+
+export async function patchSecurity(securityId, updates) {
+  const fetchOptions = {
+    method: "PATCH",
+    headers: {
+      "Content-Type": "application/json"
+    },
+    body: JSON.stringify(updates)
+  };
+  const options = await getWindowOptions();
+  const response = await fetch(
+    `${options.customData.apiBaseUrl}/securities-v2/${securityId}`,
+    fetchOptions
+  );
+  if (response.ok) {
+    return;
```

Comment:
The PATCH request returns the body of the security in its response with a 200 status so you should return it here. It's only when there's a 204 No Content, like the DELETE endpoint returns that there won't be any data to return/resolve.

Reply:
Okay cool, made the change now. Is there a reason why it needs to return the security when it doesn't get used in the component?